app/views/docs/rest.phtml

```diff
@@ -1,12 +1,310 @@
+</div>
+
+<h2>Permissions</h2>
+
+<p>Appwrite SDKs have helpers to generate permission strings, but when using Appwrite without SDKs, you'd need to create the strings yourself.</p>
+
+<h3>Permission Types</h3>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 250px">SDK</td>
+        <td>Permission String</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>Permission.read()	</code></td>
+        <td><code>"read(\"[PERMISSION_ROLE]\")"</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.create()</code></td>
+        <td><code>"create(\"[PERMISSION_ROLE]\")"</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.update()</code></td>
+        <td><code>"update(\"[PERMISSION_ROLE]\")"</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.delete()</td>
+        <td><code>"delete(\"[PERMISSION_ROLE]\")"</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.write()</td>
+        <td><code>"write(\"[PERMISSION_ROLE]\")"</code></td>
+    </tr>
+    </tbody>
+</table>
+
+<h3>Permission Roles</h3>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 250px">SDK</td>
+        <td>Role String</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>Role.any()</code></td>
+        <td><code>"any"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.guests()</code></td>
+        <td><code>"user:[ID]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.users([STATUS])</code></td>
+        <td><code>"users/[STATUS]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.user([USER_ID], [STATUS])</td>
+        <td><code>"user:[USER_ID]/[STATUS]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.team([TEAM_ID])</td>
+        <td><code>"team:[TEAM_ID]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.team([TEAM_ID], [ROLE])</td>
+        <td><code>"team:[TEAM_ID]/[ROLE]"</code></td>
+    </tr>
+    </tbody>
+</table>
+
+<ul>
+    <li class="margin-bottom"><a href="/docs/permissions" rel="noopener"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> Learn more about permissions</a></li>
+</ul>
+
+<h2>Unique ID</h2>
+<p>
+Appwrite's SDKs have a helper <code>ID.unqiue()</code> to generate unique IDs. When using Appwrite without an SDK, pass the string <code>"unique()"</code> into the ID parameter.
+</p>
+
+<h2>Query Methods</h2>
+<p>
+Appwrite's SDKs provides a <code>Query</code> class to generate query strings. When using Appwrite without an SDK, you can template your own strings with the format below.
+</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 150px">Query Method</th>
+            <th>Query String</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>equal</td>
+            <td><code>"equal(\"attribute\", [\"value1\", ...])"</code></td>
```

Comment:
`value1` should only be wrapped with double quotes if it's a string. Some examples may be helpful for different types like bools.

Reply:
Good call